security/form_login_setup.rst

```diff
@@ -373,4 +373,56 @@ deal with this low level session variable. However, the
 :class:`Symfony\\Component\\Security\\Http\\Util\\TargetPathTrait` utility
 can be used to read (like in the example above) or set this value manually.
 
+The only time target path is set from Symfony is when the user start the authentication flow, passing through the authentication entry point. This is done by the ExceptionListener, when the user tries to access a restricted page, and it is redirected to the login page. At that point target path is set.
```

Comment:
Break this long line onto multiple

Reply:
I think the second sentence makes a lot of sense, but not the first - the second is actually more accurate. Maybe just keep the second sentence?